MLnz20/README.md

```diff
@@ -0,0 +1,35 @@
+# MLnz20 
+This file is a _evaluation dataset_ of the NZ Local network magnitude MLnz20 dataset derived from offline processed Geonet Rapid event data and computed over a separate instance.
+
+It is a SeisComp _development_ magnitude implementation derived from _Rhoades et al._ (2021). 
+This computation is originally based on a GNS Seiscomp Nightly V7.0 (Development) implementation and will be upgraded over time. Results are to be considered informational for science and development purposes and subject to modifications.
+
+This dataset will be superseded as MLnz20 is implemented into GeoNet Rapid Production chain.
```

Comment:
I think I understand this bit, but might be word rephrasing that a bit for the non-experts? 
Also happy to leave as is if I'm missing something. 

The things less clear to me are (potentially not of interest for external users?):
- what you mean here with "SeisComp _development_"? if it is capturing the fact that is computed offline and independently from GeoNet Rapid, maybe you can just mention the implementation reference?
- are you able to add a link to GNS Seiscomp Nightly v7.0?
- suggest to change last sentence to something like `This dataset will be superseded once MLnz20 will be fully implemented into GeoNet Rapid Earthquake Catalogue Production system.`